### PR TITLE
Remove Bower dependency

### LINKED ISSUE #1780: Stop using Bower

### PR DESCRIPTION
**Pull request by jbrooksuk**
Closes #1780 

---

Assets need recompiling after merge.

@joecohens have I missed anything?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,8 +1,27 @@
 {
-    "dependencies": {
-        "bower": "~1.7.7",
-        "gulp": "~3.9.1",
-        "laravel-elixir": "~5.0.0"
-    },
-    "private": true
+  "name": "cachet",
+  "dependencies": {
+    "animate-sass": "git+https://github.com/tgdev/animate-sass.git",
```
Comment (joecohens) [inline package.json]: Nice!!!

**Comment by joecohens** [comment]
I don't think you did, no trouble compiling ok?

**Comment by jbrooksuk** [comment]
> I don't think you did, no trouble compiling?

None now :)

File: package.json (hunk 1 above)
Comment (jbrooksuk) [inline package.json]: We should remove this dependency for v3.0.0.

Comment (joecohens) [inline package.json]: Yeah we are not using it lol

**Comment by jbrooksuk** [comment]
Boom!

**Comment by GrahamCampbell** [comment]
image

**Pull request merged**